NCS.DSS.Interaction/GetInteractionByIdHttpTrigger/Function/GetInteractionByIdHttpTrigger.cs

```diff
@@ -38,34 +38,54 @@ public GetInteractionByIdHttpTrigger(IResourceHelper resourceHelper, IHttpReques
         [Display(Name = "Get", Description = "Ability to retrieve an individual interaction record.")]
         public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Customers/{customerId}/Interactions/{interactionId}")] HttpRequest req, string customerId, string interactionId)
         {
+            _logger.LogInformation("Function {FunctionName} has been invoked", nameof(GetInteractionByIdHttpTrigger));
+
             var touchpointId = _httpRequestMessageHelper.GetDssTouchpointId(req);
             if (string.IsNullOrEmpty(touchpointId))
             {
-                log.LogInformation("Unable to locate 'TouchpointId' in request header.");
+                _logger.LogInformation("Unable to locate 'TouchpointId' in request header.");
                 return new BadRequestObjectResult(HttpStatusCode.BadRequest);
             }
 
-            log.LogInformation("Get Interaction By Id C# HTTP trigger function  processed a request. By Touchpoint. " + touchpointId);
-
             if (!Guid.TryParse(customerId, out var customerGuid))
+            {
+                _logger.LogWarning("Unable to parse 'customerId' to a GUID. Customer GUID: {CustomerID}", customerId);
                 return new BadRequestObjectResult(customerGuid);
+            }
 
             if (!Guid.TryParse(interactionId, out var interactionGuid))
+            {
+                _logger.LogWarning("Unable to parse 'interactionId' to a GUID. Interaction ID: {InteractionId}", interactionId);
                 return new BadRequestObjectResult(interactionGuid);
+            }
```

Comment:
Need a log stating that input validation is complete, see https://github.com/SkillsFundingAgency/dss-transfer/pull/31/files#diff-695bbf80412b14a3c3dcfde0ff6fd17ff9d7119b08dcd1741520ff6e7ee8cd9f line 71 for an example